.gitignore

```diff
@@ -6,3 +6,4 @@ package-lock.json
 .cache-eslint-remote-tester
 eslint-remote-tester-results
 *.log
+.vscode
```

Comment:
https://gist.github.com/subfuzion/db7f57fff2fb6998a16c

Reply:
I'm not sure add this to global ignore is a good idea, since many repos actually stores `.vscode`.
Adding this because my VSCode keeps creating this empty directory.